packages/webcomponents/src/components/dispute-management/dispute-notification.tsx

```diff
@@ -14,48 +22,74 @@ export class DisputeNotification {
     this.submitted.emit();
   }
 
-  initiateCounterDispute() {
-    this.clickEvent.emit({ name: 'counterDispute' });
+  initiateRespondToDispute() {
+    this.clickEvent.emit({ name: DisputeManagementClickEvents.respondToDispute });
   }
 
   render() {
     return (
-      <div part={text}>
-        <h1 class="h4" part={heading3}>This payment was disputed</h1>
-        <p>The cardholder for this payment claims that the product was not received.</p>
-        <p>You can either counter the dispute by presenting proof of the product's delivery to or use by the cardholder, or you can choose to accept the dispute right away, refund the cardholder, and resolve the matter.</p>
-
-        <h2 class="h5">Dispute details</h2>
-        <div class="d-table gap-2">
-          <div class="d-table-row gap-2">
-            <span class="fw-bold d-table-cell pe-4">
-              Amount
-            </span>
-            <span class="flex-1 d-table-cell text-wrap">
-              $50.00
-            </span>
+      <div>
+        {this.isLoading && (
+          <div style={{ marginBottom: '8px' }}>
+            <Skeleton height={'28px'} width={'70%'} styles={{ marginBottom: '16px' }} />
```

Comment:
This doesn't seem to be working after resolving merge conflicts with the release branch